test/images/volume/nfs/run_nfs.sh

```diff
@@ -22,21 +22,22 @@ function start()
     while getopts "G:" opt; do
         case ${opt} in
             G) gid=${OPTARG};;
-        esac
+            *) echo "accept: -G gid = ${OPTARG}"
```

Comment:
Why did it change? What did it fix?
```
In ./test/images/volume/nfs/run_nfs.sh line 23:
        case ${opt} in
        ^-- SC2220: Invalid flags are not handled. Add a *) case.
Ref: https://github.com/koalaman/shellcheck/wiki/SC2220
```
example: 
**Problematic code**:
```
#!/bin/sh
while getopts "vr" f
do
  case "$f" in
    v) echo "verbose" ;;
    r) echo "recursive" ;;
  esac
done
```
**Correct code:**:
```
#!/bin/sh
while getopts "vr" f
do
  case "$f" in
    v) echo "verbose" ;;
    r) echo "recursive" ;;
    *) echo "usage: $0 [-v] [-r]" >&2
       exit 1 ;;
  esac
done
```

Reply:
this is not correct, it should look more like the suggested code with:
- needs to either return or exit with an error depending on the context
- usage string should be the program usage and go to stderr

Reply:
Thanks for the comments, which I'll update later.